lib/fortnox/api/models/unit.rb

```diff
@@ -0,0 +1,22 @@
+require 'fortnox/api/types'
+require 'fortnox/api/models/base'
+
+module Fortnox
+  module API
+    module Model
+      class Unit < Fortnox::API::Model::Base
+        UNIQUE_ID = :code
+        STUB = { code: '' }.freeze
+
+        # Url Direct URL to the record
+        attribute :url, Types::Nullable::String.with(read_only: true)
```

Comment:
As mentioned in #32, this attribute is actually called `@url`, but I guess we postpone that fix to when we fix #32.

Reply:
Yes.